src/plots/plot.ts

```diff
@@ -451,27 +456,42 @@ export class Plot extends Component {
    * @return {Plots.PlotEntity[]}
    */
   public entities(datasets = this.datasets()): Plots.PlotEntity[] {
-    var entities: Plots.PlotEntity[] = [];
+    var entities = this._lightweightEntities(datasets);
+
+    entities.forEach((entity) => {
+      entity.selection = entity.drawer.selectionForIndex(entity.validDatumIndex);
+      delete(entity.drawer);
+      delete(entity.validDatumIndex);
```

Comment:
I guess I mainly don't understand why `_lightweightEntities()` sets the `selection` to `null`.  This can be incredibly confusing later if developers down the line call `_lightweightEntities` expecting to have a `selection` parameter